app/controllers/api/v1/users_controller.rb

```diff
@@ -4,4 +4,28 @@ def index
 
     render json: @users
   end
+
+  # POST /users
+  def create
+    @user = User.create!(user_params)
+    json_response(@user, :created)
+  end
+
+  # DELETE /users/:id
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy if @user.present?
+    head :no_content
+  end
+
+  private
+
+  def user_params
+    # whitelist params
+    params.permit(:name, :username)
```

Comment:
In order to whitelist user attributes, name and username, you will have to require the model e.g params.require(:user).permit(:name, :username)